pages/terms-of-use.js

```diff
@@ -248,7 +248,7 @@ const DefaultTermsOfUse = ({ orgName }) => {
         that you wish to deactivate your account with us. Continued use of our
         website or Service, following notice of such changes shall indicate your
         acknowledgement of such changes and agreement to be bound by the terms
-        and conditions of such changes. Feedback You acknowledge and agrees that
+        and conditions of such changes. Feedback You acknowledge and agree that
```

Comment:
The sentence formation seems wrong. Why does the sentence start with Feedback? and there are also two capital letters in the sentence